descriptors/sqlserver/debezium-sqlserver-1.9.0.Alpha1.json

```diff
@@ -429,64 +145,6 @@
           "type" : "integer",
           "x-name" : "max.queue.size",
           "x-category" : "ADVANCED"
-        },
-        "max.queue.size.in.bytes" : {
-          "format" : "int64",
-          "title" : "Change event buffer size in bytes",
-          "description" : "Maximum size of the queue in bytes for change events read from the database log but not yet recorded or forwarded. Defaults to 0. Mean the feature is not enabled",
-          "default" : 0,
-          "type" : "integer",
-          "x-name" : "max.queue.size.in.bytes",
-          "x-category" : "ADVANCED"
-        },
-        "poll.interval.ms" : {
-          "format" : "int64",
-          "title" : "Poll interval (ms)",
-          "description" : "Time to wait for new change events to appear after receiving no events, given in milliseconds. Defaults to 500 ms.",
-          "default" : 500,
-          "type" : "integer",
-          "x-name" : "poll.interval.ms",
-          "x-category" : "ADVANCED"
-        },
-        "retriable.restart.connector.wait.ms" : {
-          "format" : "int64",
-          "title" : "Retriable restart wait (ms)",
-          "description" : "Time to wait before restarting connector after retriable exception occurs. Defaults to 10000ms.",
-          "default" : 10000,
-          "type" : "integer",
-          "x-name" : "retriable.restart.connector.wait.ms",
-          "x-category" : "ADVANCED"
-        },
-        "source.struct.version" : {
-          "title" : "Source struct maker version",
-          "description" : "A version of the format of the publicly visible source part in the message",
-          "default" : "v2",
-          "enum" : [ "v1", "v2" ],
-          "type" : "string",
-          "x-name" : "source.struct.version",
-          "x-category" : "ADVANCED"
-        },
-        "signal.data.collection" : {
-          "title" : "Signaling data collection",
-          "description" : "The name of the data collection that is used to send signals/commands to Debezium. Signaling is disabled when not set.",
-          "type" : "string",
-          "x-name" : "signal.data.collection",
-          "x-category" : "ADVANCED"
-        },
-        "transaction.topic" : {
-          "title" : "Transaction topic name",
-          "description" : "The name of the transaction metadata topic. The placeholder ${database.server.name} can be used for referring to the connector's logical name; defaults to ${database.server.name}.transaction.",
-          "default" : "${database.server.name}.transaction",
-          "type" : "string",
-          "x-name" : "transaction.topic",
-          "x-category" : "ADVANCED"
-        },
-        "database.server.timezone" : {
```

Comment:
@gunnarmorling same question here, in the Windows world, this might be an important topic, especially as even system clocks are set to local timezones often, is it? include or exclude?

Reply:
I don't think we need it for SP. We can easily add those things later on.